dev-util/jenkins-bin/jenkins-bin-2.11.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
hi
It's my first pull-request, I wasn't sure about the process.
So I preferred not to make modification on the ebuild.